### PR TITLE
fix "an operation already exists with this id"

### PR DESCRIPTION
**Pull request by kulture-rob**
#### Fixes

When triggering a new subscription while the websocket is reconnecting, GraphQL will throw an error of "an operation already exists with this id". If a subscription is adding during reconnect, the `onListen` is added to `_subscriptionInitializers` before the reconnect takes place. Then, when the connection is established, the original `onListen` triggers as well as the one that was stored off in `_subscriptionInitializers`. Since they are the same callback with the same id, an error is thrown.

### CODE REVIEW

File: packages/graphql_flutter/pubspec.yaml [hunk 1]
```diff
@@ -8,7 +8,8 @@ authors:
 - Michael Joseph Rosenthal <rosenthalm93@gmail.com>
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
 dependencies:
-  graphql: ^3.0.1-beta.1
+  graphql:
+    path: ..\graphql
```
Comment (mainawycliffe) [inline packages/graphql_flutter/pubspec.yaml]: Do not commit the path for graphql, this is for development.

Comment (kulture-rob) [reply packages/graphql_flutter/pubspec.yaml]: Ahh whoops. I'll fix that

Comment (kulture-rob) [reply packages/graphql_flutter/pubspec.yaml]: Just pushed a fix

**Comment by sokolej79** [comment]
Same issue.
It happens on Subscription widget when turn off/on internet.

**Pull request merged**